src/pickup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PICKUP_H
+#define _PICKUP_H
+
+#include <unordered_map>
+
+#include "entity.h"
+
+namespace pickupNS {
+	const enum PICKUP_TYPE { MAGNET, INVERT };
```

Comment:
What if we dont use enums and just generate an array or a map and store it as an attribute